depthai.py

```diff
@@ -402,7 +402,10 @@ def show_landmarks_recognition(entries_prev, frame):
     frame_count[s] = 0
     frame_count_prev[s] = 0
 
-entries_prev = []
+nnet_prev = {}
+nnet_prev["entries_prev"] = []
+nnet_prev["nnet_source"] = []
```

Comment:
`nnet_source` seems to be unused, maybe we can move back to `entries_prev = []`?

Reply:
see my comment below

Reply:
The logic is that `entries_prev` and `nnet_source` are tied together, that's why I created a dictionary instead 2 lists.